x-pack/legacy/plugins/integrations_manager/server/packages/install.ts

```diff
@@ -8,35 +8,40 @@ import { SavedObject, SavedObjectsClientContract } from 'src/core/server/';
 import { SAVED_OBJECT_TYPE } from '../../common/constants';
 import { AssetReference, AssetType, InstallationAttributes } from '../../common/types';
 import * as Registry from '../registry';
-import { CallESAsCurrentUser, assetUsesObjects, getInstallationObject } from './index';
+import { assetUsesObjects, getInstallationObject } from './index';
 import { getObjects } from './get_objects';
 
 export async function installPackage(options: {
   savedObjectsClient: SavedObjectsClientContract;
   pkgkey: string;
-  asset: AssetType;
-  callCluster: CallESAsCurrentUser;
 }) {
-  const { savedObjectsClient, pkgkey, asset, callCluster } = options;
-  // install any assets (in ES, as Saved Objects, etc) as required. Get references to them
-  const toSave = await installAssets({
-    savedObjectsClient,
-    pkgkey,
-    asset,
-    callCluster,
-  });
-
-  if (toSave.length) {
-    // saved those references in the package manager's state object
-    const saved = await saveInstallationReferences({
+  const { savedObjectsClient, pkgkey } = options;
+
+  // Only install Kibana assets during package installation. All other asset types need special handling
+  const assetTypes: AssetType[] = [AssetType.visualization, AssetType.dashboard, AssetType.search];
+
+  let result = {};
```

Comment:
I'm curious about the new return value. I'll run locally, but this is a reminder it'd be nice to have some tests to demonstrate/confirm the behavior.

Reply:
I've rewritten this a bit and now call `getPackageInfo()` after both installation and deletion of a package. It's another `await`, but at this point we're sure all installation steps are done, and the information contained in this response may even be more useful than just the installed objects array.